internal/bits/bit_array.go

```diff
@@ -409,6 +409,13 @@ func (bA *BitArray) UnmarshalJSON(bz []byte) error {
 	// Construct new BitArray and copy over.
 	numBits := len(bits)
 	bA2 := NewBitArray(numBits)
+	if bA2 == nil {
+		// Treat it as if we encountered the case: b == "null"
```

Comment:
The fact that `numBits = 0` is because the `bits = ""`, I'm trying to assess if the statement to that `null == ""`. In some respect maybe there's no way to generate an empty string value where `b == ""` ?

Reply:
> In some respect maybe there's no way to generate an empty string value where b == "" ?

An attacker can construct such a JSON very easily.